libraries/net/node.cpp

```diff
@@ -283,6 +193,91 @@ namespace graphene { namespace net { namespace detail {
 #define MAXIMUM_NUMBER_OF_BLOCKS_TO_HANDLE_AT_ONE_TIME 200
 #define MAXIMUM_NUMBER_OF_BLOCKS_TO_PREFETCH (10 * MAXIMUM_NUMBER_OF_BLOCKS_TO_HANDLE_AT_ONE_TIME)
 
+   /******
+    * Use information passed from command line or config file to advertise nodes
+    */
+   class list_address_builder : public node_impl::address_builder
+   {
+      public:
+      list_address_builder(fc::optional<std::vector<std::string>> address_list)
+      {
+         if (address_list.valid())
```

Comment:
IMO should treat is as an error if the list is absent or empty.

Reply:
Done